SPECS/edk2/edk2.spec

```diff
@@ -115,6 +119,11 @@ Patch0013: 0013-UefiCpuPkg-MpInitLib-fix-apic-mode-for-cpu-hotplug.patch
 Patch0014: 0014-CryptoPkg-CrtLib-add-stat.h.patch
 Patch0015: 0015-CryptoPkg-CrtLib-add-access-open-read-write-close-sy.patch
 Patch0016: 0016-OvmfPkg-set-PcdVariableStoreSize-PcdMaxVolatileVaria.patch
+%if 0%{?fedora} >= 38 || 0%{?rhel} >= 10
```

Comment:
Should we also do this for azl?

Reply:
Nope, I'll delete it and the patch that they have for it! You are right